frontend/src/app/patients/AddPatient.tsx

```diff
@@ -247,6 +250,7 @@ const AddPatient = () => {
             return phoneNumber && phoneNumber.number && phoneNumber.type;
           }
         ),
+        emails: dedupeAndCompactStrings(person.emails || []),
```

Comment:
I'm actually not sure how we want to handle submissions with blank or duplicate emails. This line will remove them - is that desired?

Reply:
yeah that seems acceptable to me